packages/scss/src/components/_actionIcon.scss

```diff
@@ -0,0 +1,20 @@
+.actionIcon {
+	background: transparent;
+	border: 0;
+	border-radius: 50%;
+	color: _color("text.default");
+	cursor: pointer;
+	height: 2rem;
```

Comment:
Shouldn't it be em rather than em here? These type of buttons are really context sensitive and needs to adapt to the size of their context